### ISSUE TITLE
Run single test from Intellij plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#56]
```
As a developer using the Intellij plugin, I would like to be able to run a 
single test.

Perhaps a means to specify the test to run via the configuration screen, or 
even better would be the 
ability to right-click on a test file and run it.

```

Original issue reported on code.google.com by `burke.we...@gmail.com` on 22 Oct 2009 at 4:37

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 27 Oct 2009 at 1:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Sergey.S...@gmail.com` on 11 May 2011 at 7:42
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
@Burke: Do you suspect to right-click on a JsTestDriver configuration file or 
on a JavaScript test file?
The first use-case works correctly starting from v1.3 of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 11 May 2011 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
This feature is implemented in 1.3.3 version of the plugin.
```

Original comment by `Sergey.S...@gmail.com` on 30 May 2011 at 12:14
- Changed state: **Fixed**
